cli/commands/rewards/generatemerkle.ts

```diff
@@ -134,3 +136,13 @@ export default class CalculateRewards extends BaseCommand {
     console.info('Done')
   }
 }
+
+function eventsJSONToArray(eventFiles: string[]): EventLog[] {
```

Comment:
What do you think about just mapping over the list of files to avoid the nesting

```typescript
return eventFiles
             .map((eventFile: string) => JSON.parse(fs.readFileSync(eventFile, 'utf8')))
             .reduce((acc, el) => acc.concat(el))

```